### PR TITLE
Northstar not launching on Steam section in Linux troubleshooting

### PR DESCRIPTION
**Pull request by itscynxx**
Came up while helping someone recently, decided I'd just add it to the wiki

### CODE REVIEW

File: docs/steamdeck-and-linux/linux-troubleshooting.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 > Read Lutris troubleshooting of [common issues with Origin](https://github.com/lutris/docs/blob/master/Origin.md) first.
 
+## Northstar not launching with Steam
+
+If you're encountering issues with Northstar launching on Steam, a very quick fix that usually works is deleting the compatadata for Titanfall2. You can do this by going to your Steam directory, going to `steamapps/compatdata`, then look for the folder called `1237970` (the App ID for Titanfall2). Deleting this folder then attempting to launch Northstar again _should_ fix the issue.
```
Comment (GeckoEidechse) [inline docs/steamdeck-and-linux/linux-troubleshooting.md]: Linebreaks in markdown source ;)

```suggestion
If you're encountering issues with Northstar launching on Steam, a very quick fix that usually works is deleting the compatadata for Titanfall2.
You can do this by going to your Steam directory, going to `steamapps/compatdata`, then look for the folder called `1237970` (the App ID for Titanfall2).
Deleting this folder then attempting to launch Northstar again _should_ fix the issue.
```

Comment (GeckoEidechse) [inline docs/steamdeck-and-linux/linux-troubleshooting.md]: I would maybe add some information what that folder contains. At least for me, when there's instructions telling me to do X (especially in regards to deleting/installing stuff) I'd like to know why/what I'm doing ^^"

Comment (GeckoEidechse) [reply docs/steamdeck-and-linux/linux-troubleshooting.md]: Did this get addressed?

Comment (itscynxx) [reply docs/steamdeck-and-linux/linux-troubleshooting.md]: > This folder contains the data used by Steam in order to make games function using Proton (assuming you're using Proton for said game).

could probably be worded better however

Comment (GeckoEidechse) [reply docs/steamdeck-and-linux/linux-troubleshooting.md]: Good enough ig

File: docs/steamdeck-and-linux/linux-troubleshooting.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 > Read Lutris troubleshooting of [common issues with Origin](https://github.com/lutris/docs/blob/master/Origin.md) first.
 
+## Northstar not launching with Steam
+
+If you're encountering issues with Northstar launching on Steam, a very quick fix that usually works is deleting the compatdata for Titanfall2.
+
+You can do this by going to your Steam directory, going to `steamapps/compatdata`, then look for the folder called `1237970` (the App ID for Titanfall2). This folder contains the data used by Steam in order to make games function using Proton (assuming you're using Proton for said game). This folder, however, sometimes has issues with Launching Northstar.
```
Comment (GeckoEidechse) [inline docs/steamdeck-and-linux/linux-troubleshooting.md]: Typo + newline
```suggestion
You can do this by going to your Steam directory, going to `steamapps/compatdata`, then look for the folder called `1237970` (the App ID for Titanfall2). This folder contains the data used by Steam in order to make games function using Proton (assuming you're using Proton for said game).
This folder, however, sometimes has issues with launching Northstar.
```

**Review by GeckoEidechse — APPROVED** [review]
Looks good now ^^

**Pull request merged**